tests/packages/Microsoft.Bot.Components.Telephony.Tests/Integration/CallTransferTests/CallTransfer_IgnoredInNonTelephonyChannel.test.dialog

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "../../../tests.schema",
```

Comment:
Shouldn't you verify that the action after the call transfer gets executed here? The action before the call transfer will always get executed in all channels, so this test will be tautological.

Reply:
From what I understood, that call transfer should not work on non telephony channel, so should I be expecting something here? perhaps I should add a AssertNoActivity?

Reply:
What is your design for the behavior in another channels? I'd assume that it is not that 'it should not work' and rather what we did with start recording, which is that in other channels, the call transfer is a no-op and we continue with the next action. If you don't have a design for this we can discuss in the telephony crew chat.

Reply:
As discussed offline, other channels should not support call transfer - Pr has also been updated accordingly